lib/src/core/implementation/servient.dart

```diff
@@ -68,9 +68,9 @@ class Servient {
 
   /// Starts this [Servient] and returns a [WoT] runtime object.
   ///
-  /// The [WoT] runtime can be used for consuming, procuding, and discovering
-  /// Things.
-  Future<WoT> start() async {
+  /// The [scripting_api.WoT] runtime can be used for consuming, procuding, and
```

Comment:
I know it is too late but I noticed a typo... opened https://github.com/eclipse-thingweb/dart_wot/pull/112